### DISCUSSION TITLE
Unable to view dimension file in console

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#837] [General]
Hi @RNarmatha @bommalla,
After the data ingestion we are not able view the dimension file in console. In Postman it is showing this error.(screenshot attached)

![Screenshot (44)](https://github.com/Sunbird-cQube/community/assets/130358614/1f6171f3-f910-4eb2-919d-939ac2c23843)

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 

Just click on the first certificate in postman and verify the certification then you can ingest the data.

**Reply by Mohanapriya777** [reply]
Hi @bommalla ,
Kindly guide me with the steps. I am not able to click the certificate in the error.

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Please follow the steps:
1. First need generate the token by using below API.
    https://<domian name>/api/ingestion/generatejwt
2. Once you generated the token in body section first certification link will be displayed.
3. Just click the link and verify the certification.
4. After that coy the bearer token and go to the new programs API and paste the token for authorisation.

**Comment by RNarmatha** [comment]
Hi @Mohanapriya777 
Every first step is generate the JWT token . Make sure it is done

**Reply by Mohanapriya777** [reply]
Hi @RNarmatha ,
We did that but still it's not working .

**Reply by RNarmatha** [reply]
Hi @Mohanapriya777 ,

Can you please share the error

**Reply by Mohanapriya777** [reply]
Hi @RNarmatha @bommalla  ,
We generated the token then also it is not showing the first certification link(screenshot attached)

![Screenshot (47)](https://github.com/Sunbird-cQube/community/assets/130358614/0da7502f-f40a-4a9c-b39d-2767fe7adea0)

**Reply by RNarmatha** [reply]
Hi @Mohanapriya777 , 

Please go to settings and enable the SSL certificate request. If you see a sign near 200, there is a warning sign. If you enabled the SSL certificate for this request, It will work. Once it is done, generate the token again.

**Reply by Mohanapriya777** [reply]
Hi @RNarmatha ,
Thank you for your support. The production domain is up and running now.
